client/src/pages/dashboard/views/Invites.svelte

```diff
@@ -50,17 +51,16 @@
 {#if currentOffice === null}
     <p>You must select an office before accessing the Invites page.</p>
 {:else}
-    <h1>Invitations</h1>
-    <Button on:click={openCreateInvite.bind(null)}>
-        <PersonAdd color={IconColor.White} size={IconSize.Normal} alt="Invite person" />Invite User
-    </Button>
-
+    <header>
+        <h1>Invitations</h1>
+        <Button on:click={openCreateInvite.bind(null)}>
+            <PersonAdd color={IconColor.White} size={IconSize.Normal} alt="Invite person" />Invite User

```

Comment:
A space between the icons is probably good here.
```suggestion
            <PersonAdd color={IconColor.White} size={IconSize.Normal} alt="Invite person" /> Invite User
```